Mixpanel.xcodeproj/xcshareddata/xcschemes/Mixpanel.xcscheme

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "1200"
+   LastUpgradeVersion = "1310"
```

Comment:
all the schemes have this change, did we want this?

Reply:
this is fine